pkgs/applications/office/onlyoffice-bin/default.nix

```diff
@@ -0,0 +1,163 @@
+{ stdenv
+, lib
+, fetchurl
+
+, alsaLib
```

Comment:
```suggestion
, fetchurl
, alsaLib
```

Reply:
Added a comment to make clearer what the gap was for.